packages/modular-scripts/src/build-scripts/webpack-scripts/config/parts/appConfig.ts

```diff
@@ -51,21 +54,24 @@ function createPluginConfig({
               }
             : undefined,
         ),
-      ),
+      ) as unknown as WebpackPluginInstance,
```

Comment:
argh, why do we need this?

Reply:
TypeScript error:
`Excessive stack depth comparing types 'HtmlWebpackPlugin' and 'WebpackPluginInstance'.ts(2321)`
Essentially it doesn't manage to check that the two are actually compatible

Reply:
Theres a few instances of this unfortunately

Reply:
Understood, can we explain it with a comment specifying `... this happens with current Typescript version <x.y.z>`? I foresee everyone coming here spending a couple of hours trying to fix this 🤣